cfgov/v1/models/ref.py

```diff
@@ -47,7 +47,7 @@
     )),
     ('Enforcement action', (
         ('fed-district-case', 'Federal District Court Case'),
-        ('admin-adj-process', 'Administrative Adjudication Process'),
+        ('admin-adj-process', 'Administrative Filing'),
```

Comment:
change `admin-adj-process` to `admin-filing`. Just checked the database and there are no pages with this category so we should be good to merge without a data migration.